src/components/Banner/Banner.scss

```diff
@@ -51,18 +52,22 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tightest);
 
 .statusSuccess {
   @include banner-colors(color(green), color(green, lighter));
+  @include icon-color(color(banner, success));
```

Comment:
Why not follow the same pattern as is used for the other colors in the banner? Eg: @include icon-color(color(green, dark));